src/valkey-cli.c

```diff
@@ -5109,6 +5363,45 @@ static int clusterManagerMigrateKeysInSlot(clusterManagerNode *source,
     return success;
 }
 
+static int clusterManagerMoveSlotRangesASM(clusterManagerNode *source, clusterManagerNode *target, list *slot_ranges, int opts, char **err) {
```

Comment:
we are sharing the same opts flags as clusterManagerMoveSlot, we need a comment in here